packages/storybook-test/stories/icon/icon.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import packageJSON from '../../../components-react/icon-react/package.json';
+import { Icon } from '../../../components-react/icon-react/src/css';
+import { Paragraph } from '../../../components-react/paragraph-react/src/css';
+
+const meta = {
+  args: {
+    children: (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="24"
+        height="24"
+        fill="none"
+        stroke="currentColor"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        strokeWidth="2"
+        viewBox="0 0 24 24"
+      >
+        <path stroke="none" d="M0 0h24v24H0z" />
+        <path d="M3 12a9 9 0 1 0 18 0 9 9 0 1 0-18 0" />
+        <path d="m10 16.5 2-3 2 3m-2-3v-2l3-1m-6 0 3 1" />
+        <circle cx="12" cy="7.5" r=".5" fill="currentcolor" />
+      </svg>
+    ),
+  },
+  argTypes: {
+    'aria-label': { table: { category: 'API' }, type: 'string' },
+    'aria-labelledby': { table: { category: 'API' }, type: 'string' },
+    // Hide children as it's a React component which cannot be displayed nicely in the Storybook UI
+    children: { table: { disable: true } },
+    className: { table: { category: 'API' }, type: 'string' },
+    dir: {
+      control: { labels: { undefined: '(undefined)' }, type: 'select' },
+      options: [undefined, 'rtl', 'ltr', 'auto'],
+      table: { category: 'API' },
+    },
+    role: {
+      control: { labels: { undefined: '(undefined)' }, type: 'select' },
+      options: [undefined, 'img'],
+      table: { category: 'API' },
+    },
+  },
+  component: Icon,
+  parameters: {
+    externalLinks: [
+      {
+        name: 'Open in Figma',
+        url: 'https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=153-1056',
+      },
+      {
+        name: 'Open op GitHub',
+        url: packageJSON.homepage,
+      },
+    ],
+  },
+  title: 'Componenten/Icon',
+} satisfies Meta<typeof Icon>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const AriaHidden: Story = {
+  name: 'Decoratief icoon met aria-hidden',
+  parameters: {
+    docs: {
+      description: {
+        story:
+          'Wanneer een icon decoratief is, is het niet nodig dat een bezoeker die gebruik maakt van hulpsoftware weet dat er een icon aanwezig is. Daarom is het nodig dat de icon aria-hidden ondersteunt.',
```

Comment:
Ik stel voor dat we de tekst bij stories niet verwoorden als acceptatiecriteria, maar als een beschrijving wat je verwacht te zien bij de story, dat als basis dient om te controleren of de story het gewenste resultaat realiseert.

Bij Code Block zijn de teksten op de manier geschreven, als je een voorbeeld wil zien.

Ook goed om dit ergens in het proces te documenteren.